hawc/apps/animal/exports.py

```diff
@@ -196,11 +196,7 @@ def _get_header_row(self):
         else:
             self.rob_headers, self.rob_data = RiskOfBias.get_dp_export(
                 self.queryset.first().assessment_id,
-                list(
```

Comment:
I tried to use the ORM on the db side to do the distinct select instead of passing the data over the wire, but I think it's less readable, what do you think?

```python
study_ids = list(
    self.queryset.values_list("animal_group__experiment__study_id", flat=True)
    .order_by("animal_group__experiment__study_id")
    .distinct("animal_group__experiment__study_id")
)
self.rob_headers, self.rob_data = RiskOfBias.get_dp_export(
    self.queryset.first().assessment_id, study_ids, "animal",
)
```

I think I prefer splitting out the study_ids, like this instead for this and other requests

```python
study_ids = set(self.queryset.values_list("animal_group__experiment__study_id", flat=True))
self.rob_headers, self.rob_data = RiskOfBias.get_dp_export(
        self.queryset.first().assessment_id, study_ids, "animal",
)
```

What do you think?

Reply:
I agree on splitting out the study_ids. And as for readability, I think the former is still acceptable; it's pretty obvious what is going on with the method chain

Reply:
ok, feel free to make the change and i think this one is ready go go!